src/main/java/frc/robot/commands/ClampCmd.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ClawSubsystem;
+
+public class ClampCmd extends CommandBase {
+
+  private ClawSubsystem claw;
+
+  public ClampCmd(ClawSubsystem claws) {
+    claw = claws;
+    addRequirements(claws);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+
+    claw.clamp();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
```

Comment:
Do these `isFinished()` calls need to be anything but `return false`?

Reply:
i dont think so because im not asking it to send anything back